<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>583</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="message">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(81, 0, 143, 67), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="placeholderText">
       <string>message</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.011, stop:0 rgba(44, 44, 44, 137), stop:1 rgba(255, 255, 255, 255));
font: 400 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>267</width>
         <height>198</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLineEdit" name="btn_name">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.511, x2:0, y2:0.534, stop:0 rgba(39, 0, 66, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="btn_L_name">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.511, x2:0, y2:0.534, stop:0 rgba(59, 0, 98, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>last name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="btn_p_number">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.511, x2:0, y2:0.534, stop:0 rgba(80, 0, 135, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>phone number</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="btn_m_number">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.511, x2:0, y2:0.534, stop:0 rgba(130, 0, 219, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>mobile number</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLineEdit" name="btn_email">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.511, x2:0, y2:0.534, stop:0 rgba(152, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>email</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="btn_create">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(174, 133, 213);
border-style: outset;
border-width: 2px;
background-color: rgb(82, 43, 132);
font: 700 22pt &quot;Georgia&quot;;</string>
      </property>
      <property name="text">
       <string>create</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="btn_delall">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(174, 133, 213);
border-style: solid;
border-width: 2px;
font: 500 14pt &quot;Georgia&quot;;
background-color: rgb(120, 0, 194);</string>
      </property>
      <property name="text">
       <string>delete all</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="btn_del">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(174, 133, 213);
border-style: solid;
border-width: 2px;
background-color: rgb(148, 0, 240);
font: 500 14pt &quot;Georgia&quot;;</string>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="btn_theme">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(174, 133, 213);
border-style: solid;
border-width: 2px;
background-color: rgb(148, 0, 240);
font: 500 14pt &quot;Georgia&quot;;</string>
      </property>
      <property name="text">
       <string>theme</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="btn_reload">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(174, 133, 213);
border-style: solid;
border-width: 2px;
font: 500 14pt &quot;Georgia&quot;;
background-color: rgb(151, 79, 240);</string>
      </property>
      <property name="text">
       <string>reload</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>304</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
